src/infrastructure/repository/ride/MongoRideMapper.js

```diff
@@ -0,0 +1,25 @@
+const Ride = require('../../../domain/ride/Ride');
+const User = require('../../../domain/user/User');
+
+const MongoRideMapper = {
+  toEntity(dataValues) {
+    const { driver, passengers, date } = dataValues;
+
+    return new Ride({
+      driver: new User(driver),
+      passengers: passengers.map(passenger => new User(passenger)),

```

Comment:
Hum... passengers may be called users ?

Reply:
(it seems you change the "type" of the passeger by doing new User(passenger) ? is not it ?

Reply:
Later in the project, i'll have 2 classes : Driver and Passangers (because, later, they'll have their own attributes). Right now, YAGNI, I just need one user type... So... KISS, i decided to stick with only 1 class for now. Is it bad?

If my variable is called "passenger", it's for code readability :) I'm actually manipulating a passenger :)